packages/meditrak-server/src/routes/utilities/hasAccessToEntityForVisualisation.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+export const hasAccessToEntityForVisualisation = async (
```

Comment:
I believe this won't be needed as a util after all, but can instead just be used within the dashboard group permissions checker

Reply:
I still keep this cause it's used for both dashboard groups and map overlays